src/bundle/Resources/views/themes/admin/ui/menu/content_edit_menu.html.twig

```diff
@@ -0,0 +1,63 @@
+{% extends '@KnpMenu/menu.html.twig' %}
```

Comment:
This menu will be used in different places not only in content edit. It should be named differently.